### ISSUE TITLE
Allow ES Search JSON to be prettyfied

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#8958]
If you run a query via the ES Search api the result comes back as a blob:
```
curl -H "Content-Type: application/json" -XPOST http://localhost:8080/api/es/search -d '{
   "query": {
      "filtered": { 
         "query": {
            "query_string" : {
               "query" : "+news.title:retirement"
            }
            },
            "filter": {
               "range": { "news.sysPublishDate": { "gt": "20160128" }}
            }
         }
      }
   }
}
'
```

It would be nice to allow developers a switch to prettify the output by passing a flag like ?pretty=1 or something.

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/9083

**Comment by bryanboza** [comment]
Fixed, tested on master-3.6 // Postgres // FF

**Comment by john-thomas-dotcms** [comment]
Doc Link: http://auth.dotcms.com/docs/latest/es-restful-services#Pretty
